sqlc.yaml

```diff
@@ -0,0 +1,10 @@
+version: "2"
+sql:
+  - engine: "postgresql"
```

Comment:
q: sqlc also has a sqlite engine, would there be any benefit in compiling to 2 different engines? I would guess that would mean we need to maintain more queries / migrations

Reply:
I believe the `sqlite` engine has some bugs that justify using the `postgresql` engine. Just need to make sure that the generated code is indeed `sqlite` compatible.

Reply:
We've been using the postgres engine fine with both sqlite and postgres. I think you only need two if you want very specific queries that can't easily be generalizable. Then at runtime, the invoice manager can do a type assertion for a more specific interface (like batch insertion with postgres), then use that.

Reply:
Yes, added as a comment in the first commit.

```
Because we plan to support sqlite and postgres with (almost) the same
sql code we should aim to write sql code that can be executed by both
engines.

The tool needs to be configured for one specific engine, in this case
postgres but the generated code can be executed by both.

In some specific cases, we will `match and replace` some sql lines to be
sure the queries are valid for the target engine.
```